packages/vaadin-login/test/login-form.test.js

```diff
@@ -187,21 +186,21 @@ describe('login form', () => {
     enter(vaadinLoginPassword);
     expect(submitStub.called).to.be.false;
 
-    tap(submit);
+    click(submit);
```

Comment:
Probably, it is a mistake because `tap` !== `click`

Reply:
I think we can use `click` here as the login does not use `tap` handlers, unlike `vaadin-date-picker`.